code.py

```diff
@@ -43,8 +44,9 @@ def __init__(self, bmp_folder: str = "/bmp", gif_folder: str = "/gif", skip_conn
         # Dictonary order is irrelevant, it will be sorted when switching apps
         self.apps = {
             LoopImagesApp.name: lambda: LoopImagesApp(self.bmp_file_list, self.display),
-            ClockApp.name: lambda: ClockApp(self.display, self.network, self.real_time),
+            ClockApp.name: lambda: ClockApp(self.display, self.real_time),
             GifPlayerApp.name: lambda: GifPlayerApp(self.gif_file_list, self.display),
+            AnalogueClockApp.name: lambda: AnalogueClockApp(self.display, self.real_time),
```

Comment:
Nitpicking, and you don't have to touch it in this PR, but it bothers me a little that `self.display` is not always the first param 🙈